root/programs/C#/Frameworks/Infrastructure/CustomControl/Control/WebCustomTextBox.cs

```diff
@@ -326,12 +326,12 @@ public bool Validate(out string[] result)
             if (this.CheckType != null)
             {
                 // 必須入力チェック
-                if (this.CheckType.IsIndispensabile)
+                if (this.CheckType.IsIndispensable)
                 {
                     if ((text == ""))
                     {
                         hasError = true;
-                        lstRet.Add(CmnCheckFunction.IsIndispensabileCheckErrorMessage);
+                        lstRet.Add(CmnCheckFunction.IsIndispensableCheckErrorMessage);
```

Comment:
@daisukenishino 

I think that the field "lstRet" is unnecessary.
- The type of argument "result" should be List<string>
- ErrorMessage should be added in "result" directly.